wasm/src/std.h

```diff
@@ -10,4 +10,16 @@
 void opa_abort(const char *msg);
 void opa_println(const char *msg);
 
+#ifdef DEBUG
+#include "printf.h"
+#define TRACE(...)                               \
```

Comment:
I'd wrap a multi line macro contents in do - while as follows:

#define TRACE(...) \
   do {
      char __trace_buf ...
   } while (0);

This guarantees you can use the macro anywhere.

Reply:
Thats good to know 👍   Will update it.